delphi-frontend/src/main/java/au/com/integradev/delphi/operator/OperatorInvocableCollector.java

```diff
@@ -270,58 +320,129 @@ private Set<Invocable> createAnd() {
     Type uint32 = typeFactory.getIntrinsic(IntrinsicType.CARDINAL);
     Type uint64 = typeFactory.getIntrinsic(IntrinsicType.UINT64);
 
+    Type widenedUInt16 = ((TypeFactoryImpl) typeFactory).widenedUInt16();
+    Type widenedUInt32 = ((TypeFactoryImpl) typeFactory).widenedUInt32();
```

Comment:
I don't understand the names here.
It doesn't make sense as a UInt32 that has been widened, because it's a smaller range than that of UInt32.

Reply:
Agreed, these names are a little awkward.

The idea was to express:
- "unsigned integer that has been widened to a 16-bit value"
and
- "unsigned integer that has been widened to a 32-bit value"

I don't love the current names, but also don't have many alternate ideas.

Thoughts?

Reply:
The host type for the `0..High(Integer)` type is `Integer`, so it feels wrong to put `UInt` in the name of these.
Even if it is _semantically_ better to think of these as unsigned integers, the width given doesn't make sense; the `widenedUInt32` type doesn't have 32 usable bits, it has 31.

I'm also not sure what the best names for these are, but I feel like the current names are misleading in more than one way:
* what exactly is being 'widened' is not clear
* the width in the name doesn't correspond to the width of the actual type

What do you think about `anonymousUInt15` and `anonymousUInt31`?

Reply:
Thanks! I went with `anonymousUInt*`